services/harmony/app/util/parameter-parsers.ts

```diff
@@ -121,4 +165,44 @@ export function handleFormat(
       .map((v: { mimeType: string }) => v.mimeType)
       .filter((v) => v);
   }
-}
\ No newline at end of file
+}
+
+/**
+ * Handle the height parameter in a Harmony query, adding it to the DataOperation
+ * if necessary.
+ *
+ * @param operation - the DataOperation for the request
+ * @param query - the query for the request
+ * @param req - the request
+ */
+export function handleHeight(
+  operation: DataOperation,
+  query: Record<string, string>): void {
+  if (query.height) {
+    try {
+      operation.outputHeight = parseNumber(query.height);
+    } catch (e) {
+      convertParameterParsingError(e, 'height');
+    }
+  }
+}
+
+/**
+ * Handle the width parameter in a Harmony query, adding it to the DataOperation
+ * if necessary.
+ *
+ * @param operation - the DataOperation for the request
+ * @param query - the query for the request
+ * @param req - the request
```

Comment:
No `req` param

Reply:
Good catch. I originally had one, then realised it wasn't being used. Will clean up now.